content/departments/cloud/technical-docs/v2.0/delete_process.md

```diff
@@ -70,39 +70,27 @@ cd environments/$ENVIRONMENT/deployments/$INSTANCE_ID/
 npx --yes cdktf-cli@0.13.3 deploy tfc
 ```
 
-### Remove namespace
+### Remove workload
 
 ```sh
-mi2 instance workon -e $ENVIRONMENT --slug $SLUG -exec
-
+# gracefully remove unmanaged resources, e.g. GKE Backup
+mi2 instance destroy --slug $SLUG -e $ENVIRONMENT -auto-approve
 # deletes namespace and Network Endpoint Group Health check
-kubectl delete ns $NAMESPACE
+mi2 instance workon -e $ENVIRONMENT --slug $SLUG -exec
+kustomize build --load-restrictor LoadRestrictionsNone --enable-helm . | kubectl delete -f -
 ```
 
 ### Disable delete protection
 
 ```sh
 # delete sql protection
+mi2 instance edit --query '.spec.debug.enableDeletionProtection = false' --slug $SLUG -e $ENVIRONMENT
+mi2 generate cdktf --slug $SLUG -e $ENVIRONMENT
 cd environments/$ENVIRONMENT/deployments/$INSTANCE_ID/terraform/stacks/sql
 terraform init
-export SQL_RESOURCE=$(terraform state list | grep sql_self)
-echo "$(jq '.resource.'$SQL_RESOURCE' += {"delete_protection":false}' cdk.tf.json)" > cdk.tf.json
 terraform apply -auto-approve
 ```
 
-### Removes GKE backups and restores
```

Comment:
We don't need to remove these anymore?

Reply:
> We don't need to remove these anymore?

these steps was extremely error-prone, so we replace it with `mi2 instance destroy --slug $SLUG -e $ENVIRONMENT -auto-approve`

https://github.com/sourcegraph/handbook/pull/5724/files#diff-7b1d9ed5a07c946916c213841da752199a404bf3cb978d9d6d795154b151f942R76-R77